lib/harvest_notifier/slack.rb

```diff
@@ -18,6 +18,11 @@ def post_message(body)
       self.class.post("/chat.postMessage", body: body)
     end
 
+    def update_message(body, update_url)
+      self.class.headers.delete "Authorization"
```

Comment:
may be add `self.class.headers "Authorization" => "Bearer #{token}"` in methods when we need instead of declaring in `initialize` method?

Reply:
Fixed in 39a0c0a315960ce354d585b91313ee7ad3aa93c0